### DISCUSSION TITLE
When do toots from other instances reach your instance?

### DISCUSSION THREAD

**Discussion by simonldwg** [mastodon/mastodon#26625] [Q&A] [answered]
Under which circumstances do toots from other instances reach your own instance?

Here are all the different ways I know so far:
* Toots reach your instance when someone from your instance follows the account that posted the toot on the other instance
* Relays populate your instance with content from other instances

Are there any more ways a toot can reach your instance and show up in the federated timeline? 

Also, the posts that show up when viewing a specific hashtag were also originally received for one of these reasons, correct?

Thank you!

**Comment by renchap** [comment, marked as answer]
- someone on your instance replies to the post
- or boosts the post
- or search for the post (with the post URL in the search bar)
- maybe other ways?

And yes, your instance only displays the posts it knows about, including when viewing a hashtag
